client/scss/layouts/_login.scss

```diff
@@ -23,32 +33,60 @@ $desktop-nice-padding: 100px;
   }
 
   h1 {
-    font-weight: 300;
-    font-size: 2em;
-    line-height: 1em;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
     white-space: nowrap;
+    font-weight: 800;
+    font-size: 30px;
+    line-height: 130%;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex: none;
+    flex-grow: 0;
+    z-index: 1;
+    margin: 0px;
   }
 
   form {
-    width: 100%;
-
-    ul {
-      @include unlist();
-    }
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 50px 60px 40px;
+    gap: 30px;
+    isolation: isolate;
+    background: #ffffff;
+    box-shadow: 5px 5px 30px rgba(0, 0, 0, 0.25);
+    border-radius: 10px;
+    flex: none;
+    order: 6;
+    flex-grow: 0;
   }
 
   label {
     width: auto;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
+    font-weight: 500;
+    font-size: 14px;
   }
 
   .button,
   a.button {
-    line-height: 1.2em;
-    font-size: 1.5em;
-    padding: 1.1em 2.4em;
-    height: 3.5em;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 18px;
+    gap: 10px;
+    width: 340px;
```

Comment:
Be careful declaring fixed widths and even fixed heights - it may come back to cause issues. 

Specially the width. 

Think about your layout as going outside in or inside out. And width 100% filling up the gaps. 

This field will likely overflow on a small device with < 400px width.